client/query.go

```diff
@@ -103,18 +99,12 @@ func (ctx Context) queryABCI(req abci.RequestQuery) (abci.ResponseQuery, error)
 		return result.Response, nil
 	}
 
-	if err = ctx.verifyProof(req.Path, result.Response); err != nil {
```

Comment:
so proofs are not verified anymore? I don't think this is desirable

Reply:
the lite.verifier is no longer present in tendermint. These methods are no longer useable.

Reply:
The context is still using `tendermint/lite/proxy` - I don't know if that's up to date or not

Reply:
If I'm understanding this correctly, this removes verification of the proof of the internal query returned from a sdk application process to the querying client. This is technically fine because these queried results shouldn't be used to execute any state. If one is updating state they should be using the light client not this verifier. 

Maybe we should update documentation to explain this?

Reply:
Is the verification just being pushed downstream to the light client? Can one query through the light client?

Reply:
From my understanding, this is used if the query is being executed against a distrusted node. 

I will go through the docs and update the removal of this feature. In the future, I believe operators that do not wish to run a full node will be able to run a light client if they would like verification. This can be done in 0.40.x as it should not introduce any breaking changes

Reply:
> Is the verification just being pushed downstream to the light client? Can one query through the light client?

there is no notion of running a light client in the Cosmos-SDK from my limited knowledge.

Reply:
gotcha, so basically

* trust no one -> full node
* trust someone -> light client
* trust many folk -> use a service

Reply:
> gotcha, so basically
>trust no one -> full node
>trust someone -> light client
>trust many folk -> use a service

ok this makes sense then

Reply:
note that the binary (full-node) will connect and serve queries through itself by default and not need to prove anything. If you override this behavior by specifying `--node=...` when you do a query, then you'll need to potentially verify the proof. To do this, you'll need a light client.

Honestly, I'm not sure about the UX. We can leave the `--node` flag, but then maybe we stipulate that you cannot trust nodes in this case and that you should start a light client.